docs/docs/noir/standard_library/bigint.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Big Integers
+description: How to use big integers from Noir standard library
+keywords:
+  [
+    Big Integer,
+    Noir programming language,
+    Noir libraries,
+  ]
+---
+
+The BigInt module in the standard library exposes some class of integers which do not fit (well) into a Noir native field. It implements modulo arithmetic, modulo a 'big' prime number.
+Currently only 6 class of integers (i.e 'big' prime numbers) are available throughout the module, namely:
+
+- BN254 Fq: Bn254Fq
+- BN254 Fr: Bn254Fr
+- Secp256k1 Fq: Secpk1Fq
+- Secp256k1 Fr: Secpk1Fr
+- Secp256r1 Fr: Secpr1Fr
+- Secp256r1 Fq: Secpr1Fq
```

Comment:
Is there one that is recommended out of the six?

As in if I am a developer that knows nothing about curves and just want one big integer type to stuff things into, which one should I use?

Reply:
You should use the Noir native field.
These big integers are useful for implementing cryptographic primitives on other curves, so you should know about them if you want to use them.
I think what you ask for is u256

Reply:
Oh maybe I'm misunderstanding things.

Is the intention of bigint here not necessarily to add an arbitrarily-sized integer type?